data/CodingSpecies.json

```diff
@@ -15,4 +16,41 @@
       "icon": "link"
     }
   ]
+  "milestones": [
```

Comment:
Invalid json, profile broken, I fixed here https://github.com/EddieHubCommunity/LinkFree/commit/062c4dd3121736ab6cdf95afc1feb24cb43cc8e2

Reply:
Sorry, I didn't see the comma errors. Thank you for fixing them!